src/style.css

```
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+  
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 600;
+  font-style: normal;
+}
+  
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+  
+.srchForm {

```

```suggestion
.searchForm {

```
Думаю можно не сокращать, две буквы погоды не сделают